prac_10/testing.py

```diff
@@ -22,7 +22,20 @@ def is_long_word(word, length=5):
     >>> is_long_word("Python", 6)
     True
     """
-    return len(word) > length
+    return len(word) >= length
+
+
+def is_formatted(phrase):
```

Comment:
This should not be a Boolean (is) function, but one that converts a phrase to a sentence form. Please read the question again, and adjust.